tekton/integration_pipeline.go

```diff
@@ -194,40 +193,52 @@ func getPipelineRunYamlFromPipelineRunResolver(client client.Client, ctx context
 		return "", err
 	}
 
+	fmt.Printf("resolution request: %+v\n", request)
```

Comment:
This looks like a line added for debugging, should we replace this with a logger, or remove it altogether?

Reply:
I'll remove it